UnitsNet.Tests/QuantityInfoTest.cs

```diff
@@ -15,18 +15,17 @@ public class QuantityInfoTest
         public void Constructor_AssignsProperties()
         {
             var expectedZero = Length.FromCentimeters(10);
-            var expectedUnits = new Enum[] {LengthUnit.Centimeter, LengthUnit.Kilometer};
-            var expectedBaseUnit = LengthUnit.Meter;
+            var expectedUnitInfos = new UnitInfo[]{ new UnitInfo(LengthUnit.Centimeter, new BaseUnits(LengthUnit.Centimeter)), new UnitInfo(LengthUnit.Kilometer, new BaseUnits(LengthUnit.Kilometer))};
```

Comment:
Would it make sense to add a more convenient `UnitInfo` ctor that take a single enum value and assigns `BaseUnits` with that value unless otherwise specified? It seems a very common call pattern in this PR.

Reply:
No - that would only be applicable for base quantities (length, mass, etc.), and would require more code than necessary for an info holder (check for base dimensions being a base quantity, throwing exceptions if unable to map, case statement for applicable assignment, etc.)

Derived quantities have no applicable mapping.